SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -2,22 +2,32 @@
 
 #include "Utils/SpatialDebugger.h"
 
+
+#include "Interop/SpatialReceiver.h"
+#include "Interop/SpatialStaticComponentView.h"
+#include "LoadBalancing/WorkerRegion.h"
+#include "Schema/AuthorityIntent.h"
+#include "SpatialCommonTypes.h"
+#include "Utils/InspectionColors.h"
+
 #include "Debug/DebugDrawService.h"
 #include "Engine/Engine.h"
 #include "EngineClasses/SpatialNetDriver.h"
 #include "GameFramework/Pawn.h"
 #include "GameFramework/PlayerController.h"
 #include "GameFramework/PlayerState.h"
-#include "Interop/SpatialReceiver.h"
-#include "Interop/SpatialStaticComponentView.h"
 #include "Kismet/GameplayStatics.h"
-#include "Schema/AuthorityIntent.h"
-#include "Utils/InspectionColors.h"
+#include "Net/UnrealNetwork.h"
 
 using namespace SpatialGDK;
 
 DEFINE_LOG_CATEGORY(LogSpatialDebugger);
 
+namespace
+{
+	const FString DEFAULT_WORKER_REGION_MATERIAL = TEXT("/SpatialGDK/SpatialDebugger/Materials/TranslucentWorkerRegion.TranslucentWorkerRegion");
```

Comment:
same comment as hardcoded asset paths above

Reply:
So currently in order for the material color and opacity changing c++ code to work we need to have a material with params named as `Color` and `Opacity` explicitly. Original plan was to expose the material so people could change if need be but ultimately I think this would just end up trolling people hence have it just constant for now.